app_dart/Dockerfile

```diff
@@ -5,7 +5,7 @@
 
 # The VM service listens on port 8181
 
-FROM google/dart-runtime:dev
+FROM google/dart-runtime@sha256:401b118e8fe9b791129d156f0918c740d441e94a395c8b37167315a4318ace6f
```

Comment:
Last time we did this, we saw AppEngine failing. Can we do a test version deploy to make sure Dart is able to start on these hashes

Reply:
Sure, I can try to get a test deploy up. Is [this the process](https://github.com/flutter/cocoon#deploying-a-test-version-on-google-cloud) which would validate the change?

Reply:
That should work, but to confirm it would be good to deploy it to appengine as well (which is in https://g3doc.corp.google.com/company/teams/flutter/infrastructure/cocoon/local_development.md?cl=head). \cc @keyonghan

Reply:
Let's keep `dev` and add the hash after it.

The previous failure was caused by migrating to the new image in docker instead of the one in google registry.

Reply:
@CaseyHillers Deployed test to appengine and can successfully access web UI